shell/client/admin/user-details-client.js

```diff
@@ -231,17 +256,22 @@ Template.newAdminUserDetails.events({
     });
   },
 
-  "click .delete-account"(evt) {
-    const instance = Template.instance();
-    const confirm = window.confirm("Really delete this account forever? This operation is permament and irreversible.");
-    if (confirm) {
-      Meteor.call("deleteAccount", instance.userId, function (err) {
-        if (err) {
-          window.alert(err);
-        } else {
-          Router.go("newAdminUsers");
-        }
-      });
-    }
+  "click [name=\"delete-account\"]"(evt, instance) {
+    instance.showDeletePopup.set(true);
+  },
+
+  "click [name=\"cancel-delete-account\"]"(evt, instance) {
+    instance.showDeletePopup.set(false);
+  },
+
+  "click [name=\"delete-account-real\"]"(evt, instance) {
+    instance.deleteSubmitting.set(true);
+    Meteor.call("deleteAccount", instance.userId, function (err) {
+      if (err) {
+        instance.deleteError.set(err);
```

Comment:
Set this to `err.message` or an appropriate field, since you appear to just be passing it through to the template.   Otherwise it'll render as `[object Object]`